labellab-client/src/actions/image.js

```diff
@@ -40,10 +40,14 @@ export const submitImage = (data, projectId, callback) => {
   }
 }
 
-export const fetchProjectImage = (imageId, callback) => {
+export const fetchProjectImage = (projectId, imageId, callback) => {
   return dispatch => {
+    if (imageId === undefined) {
```

Comment:
This seems to be caused when the image is not saved in the database (crop phase) so will not be able to fetch the file from the database without image_id.

Reply:
My idea was to only check if it is not initialized, as `!imageId` would be true for `0`. The option best would be to replace with `Number.isInteger()` to be more robust. Will do the change.

Reply:
Id cannot be zero as MySQL starts naming ids for 1. Regardless, your update solution works so it's fine.

Reply:
Yeah, that is true as well :) Are any more changes required?